operator/model/src/main/java/io/apicurio/registry/operator/api/v1/spec/ComponentSpec.java

```diff
@@ -98,4 +98,17 @@ public IngressSpec withIngress() {
         }
         return ingress;
     }
+
+    /**
+     * Indicates whether to create a pod disruption budget
+     */
+    @JsonProperty("manageDisruptionBudget")
+    @JsonPropertyDescription("""
+            Whether a PodDisruptionBudget should be managed by the operator.  Defaults to 'true'.
+
+            Set this to 'false' if you want to create your own custom PodDisruptionBudget.
+            """)
+    @JsonSetter(nulls = Nulls.SKIP)
+    private Boolean manageDisruptionBudget;
```

Comment:
I kind of prefer
```
spec:
  app:
    podDisruptionBudget:
      enabled: false
```
more than
```
spec:
  app:
    manageDisruptionBudget: false
```
there would be place for more config under `spec.app.podDisruptionBudget`, similar to `spec.app.ingress`. WDYT?

Reply:
I'm fine with this, but when we chatted about it we didn't think there would ever be a need for "more config" for PDBs and NetworkPolicies.  The whole idea was to have smart, managed resources for them.  And if the user wanted to customize them, then they could disable them and do their own thing.

That's why I did what I did.

But I'm totally fine with this alternate approach.

Reply:
True, in that case maybe `enablePodDisruptionBudget`? BTW I'm working on addressing the test problem.

Reply:
I changed it as suggested.  Note that the default is still `enabled=true` which might be confusing.  But it seems reasonable as-is.